### ISSUE TITLE
gnt-cluster modify --shared-file-storage-dir not working

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#867]
Originally reported of Google Code with ID 811.

```
(submitting this as an extra bug as 713 was side-tracked with it.)

<b>What steps will reproduce the problem?</b>
1. See the discussion in https://code.google.com/p/ganeti/issues/detail?id=713

gnt-cluster modify would not accept the option --shared-file-storage-dir

```
Originally added on 2014-04-22 11:08:52 +0000 UTC.

Mentioned Issue with GoogleCode ID 713 has been migrated to GitHub with ID #769.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Thank you Helga :)
```
Originally added on 2014-04-22 11:18:12 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 8c7864ba667af57a1f9fc7bed1acb922708daef8
Author: Helga Velroyen <helgav@google.com>
Date:   Tue Apr 22 13:00:35 2014 +0200

    Hooking up verification for shared file storage
    
    As for the cluster modify, it was also forgotten to
    hook up the verification of the shared file storage
    paths despite all infrastructure was done already.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>

commit c02767f0279b1d65e25c3d797145771e5428eb2b
Author: Helga Velroyen <helgav@google.com>
Date:   Tue Apr 22 12:51:07 2014 +0200

    Fix --shared-file-storage-dir option of gnt-cluster modify
    
    While all infrastructure to make shared-file storage
    runtime-configurable was already submitted, the actual
    setting of the path was forgotten. This patch fixes it.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Klaus Aehlig <aehlig@google.com>



```
Originally added on 2014-04-22 14:03:39 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released in 2.10.4.
```
Originally added on 2014-05-15 10:36:57 +0000 UTC.

Changed State: Released
